cfgov/v1/util/util.py

```diff
@@ -68,6 +68,9 @@ def get_secondary_nav_items(request, current_page):
     else:
         page = get_appropriate_page_version(request, current_page)
 
+    if not page:
```

Comment:
This logic change is necessary because when previewing a `BrowsePage` the value of `page` is always `None`.